src/hotspot/share/nmt/nmtTreap.hpp

```diff
@@ -382,7 +384,8 @@ class Treap {
         head = head->left();
       }
       head = to_visit.pop();
-      f(head);
+      if (!f(head))
+        return;
```

Comment:
Style: Always use braces in `if` statements.

Reply:
Done.